local-registry/Cargo.toml

```diff
@@ -175,40 +141,27 @@ proptest = "1.2.0"
 queues = "1.1.0"
 quickcheck = "1.0.3"
 quickcheck_macros = "1.0.0"
-ramp = "0.7.0"
 rand = "0.8.5"
 rand_chacha = "0.3.1"
 rayon = "1.7.0"
-readonly = "0.2.10"
 recur-fn = "2.2.0"
-reduce = "0.1.5+deprecated"
 regex = "1.9.1"
 reikna = "0.12.3"
 rs_poker = "3.0.0-alpha.5"
 rstest = "0.18.1"
 rstest_reuse = "0.6.0"
 rug = "1.19.2"
-rusqlite = "0.29.0"
 rustc-hash = "1.1.0"
-rustversion = "1.0.14"
-say-number = "1.0.0"
 scan_fmt = "0.2.6"
 scoped-pool = "1.0.0"
 scoped_threadpool = "0.1.9"
 serde = "1.0.173"
 serde_derive = "1.0.173"
-serial_test = "2.0.0"
-simplelog = "0.12.1"
-simple_parallel = "0.3.0"
-sliding_windows = "3.0.0"
```

Comment:
This could maybe be useful for a couple of exercises like series?

Reply:
It's definitely useful, but again it's also unmaintained (last release about 6 years ago). The same functionality is provided by the much more popular and maintained crate [itertools](https://docs.rs/itertools/latest/itertools/).